Add button with Form to add Wishlists

RaviN opened the pull request:
Used Bootstrap Lightbox ( http://jbutz.github.com/bootstrap-lightbox/ ) with some slight modifications to get this going.

RaviN committed:
Added lightbox with form for adding lists

carmelyne commented:
The button doesn't work after the full load of masonry.js. Let's figure this out.

RaviN commented:
Hmm that's odd, I'm on it

RaviN committed:
fixed issue where lightbox stopped working

Had to hack the lightbox library to register the lightbox to our add
button specifically, masonry was somehow messing with the data of
different elements on the page. All good now.

RaviN commented:
It should be fixed now.
